tg/views/__init__.py

```diff
@@ -129,32 +129,43 @@ def resize(self, p: float = 0.25) -> None:
         self.win.resize(self.h, self.w)
 
     def _msg_color(self, is_selected: bool = False) -> int:
-        return get_color(white, -1) | (reverse if is_selected else 0)
+        color = get_color(white, -1)
+        if is_selected:
```

Comment:
why did you remove inline selection? `| 0` has no any effects. its logical OR with 0, so left operat should always be preserved.
``` 
> get_color(white, -1)
512
> get_color(white, -1) | 0
512
```

Reply:
I think it's more readable and there are less or operations

Reply:
okey, feel free to merge then